### ISSUE TITLE
Failed to import ktx data for LVV/Horta

### ISSUE THREAD

**Issue by freshpomelo** [JaneliaSciComp/jacs-cm#16]
Hi everyone,

I follow the instruction of "MouseLight Deployment" to generate the octree and ktx data.
The tree of octree and ktx is:
![tree](https://user-images.githubusercontent.com/10544528/71503447-277bc780-28b0-11ea-9b1a-29adde0105f1.png)

My transform.txt is as follow:
```
ox: 10000
oy: 10000
oz: 10000
sx: 2048.0
sy: 2048.0
sz: 20.0
nl: 2
```
But when I import the octree, there is an info displayed:
<img width="786" alt="warning" src="https://user-images.githubusercontent.com/10544528/71502911-c6eb8b00-28ad-11ea-8179-fb119cab48e9.png">

I find this warning is caused by the attribute "filesystemSync" of class TmSample.

Still the workstation could display the 2d imagery successfully.

But when I try to show the 3d imagery from ktx data, there is an error:
<img width="496" alt="error" src="https://user-images.githubusercontent.com/10544528/71502943-f1d5df00-28ad-11ea-81c2-7949ff261f48.png">

I don't know how to set the ktx data as the "synch source".

Any advice would be greatly appreciated!

**Comment by freshpomelo** [comment]
I compare the log of docker jacs_jacs-sync with source code of jacs-compute.
The log is:
```
2019-12-27_13:28:28.942 [XNIO-1 task-181] INFO  o.j.j.r.s.v.d.TmSampleResource [service=] [rootService=] - Creating new TmSample dulab_test2 with path /jade1/dulab_test2
2019-12-27_13:28:29.351 [XNIO-1 task-181] INFO  o.j.j.d.s.DataStorageLocationFactory [service=] [rootService=] - Create JADE volume location with URLs http://dulabyoumin-mac.local:9881/jacsstorage/agent_api/v1, http://dulabyoumin-mac.local:9881/jacsstorage/agent_api/v1/agent_storage/storage_volume/2748132569525522445 and volume path dulab_test2
2019-12-27_13:28:29.354 [XNIO-1 task-181] INFO  o.j.j.r.s.v.d.TmSampleResource [service=] [rootService=] - Reading transform.txt from http://dulabyoumin-mac.local:9881/jacsstorage/agent_api/v1/agent_storage/storage_volume/2748132569525522445/dulab_test2/
2019-12-27_13:28:29.499 [XNIO-1 task-181] INFO  o.j.j.r.s.v.d.TmSampleResource [service=] [rootService=] - Found 2 levels in octree
2019-12-27_13:28:29.500 [XNIO-1 task-181] INFO  o.j.j.r.s.v.d.TmSampleResource [service=] [rootService=] - KTX data path not provided for dulab_test2. Attempting to find it relative to the octree...
2019-12-27_13:28:29.625 [XNIO-1 task-181] INFO  o.j.j.d.s.DataStorageLocationFactory [service=] [rootService=] - Create JADE volume location with URLs http://dulabyoumin-mac.local:9881/jacsstorage/agent_api/v1, http://dulabyoumin-mac.local:9881/jacsstorage/agent_api/v1/agent_storage/storage_volume/2748132569525522445 and volume path dulab_test2/ktx
2019-12-27_13:28:29.625 [XNIO-1 task-181] INFO  o.j.j.r.s.v.d.TmSampleResource [service=] [rootService=] - RAW data path not provided for dulab_test2. Attempting to read it from the tilebase.cache.yml...
2019-12-27_13:28:29.693 [XNIO-1 task-181] INFO  o.j.model.access.domain.DomainDAO [service=] [rootService=] - Updated permissions for 1 entities
2019-12-27_13:28:29.706 [XNIO-1 task-181] INFO  o.j.model.access.domain.DomainDAO [service=] [rootService=] - Updated permissions for 1 entities
2019-12-27_13:28:29.710 [XNIO-1 task-181] INFO  o.j.j.r.s.v.d.TmSampleResource [service=] [rootService=] - Saved new sample as TmSample#2750428729770508300
2019-12-27_13:28:29.722 [XNIO-1 task-181] INFO  o.janelia.jacs2.app.JAXSyncAppConfig [service=] [rootService=] - 172.22.0.3 youmin youmin 'Workstation' PUT http://dulabyoumin-mac.local/api/rest-v2/mouselight/data/sample ?subjectKey=user%3Ayoumin status=201 response_bytes=418 rt=0.93 tp=0.004 'Jersey/2.21 (HttpUrlConnection 1.8.0_231)' {"subjectKey":"user:youmin","references":null,"objectIds":null,"ordering":null,"objectType":null,"propertyName":null,"propertyValue":null,"domainObject":{"class":"org.janelia.model.domain.tiledMicroscope.TmSample","name":"dulab_test2","ownerKey":null,"readers":[],"writers":[],"creationDate":null,"updatedDate":null,"micronToVoxMatrix":null,"voxToMicronMatrix":null,"origin":null,"scaling":null,"numImageryLevels":null,"files":{"LargeVolumeOctree":"/jade1/dulab_test2"},"filesystemSync":true,"filepath":null,"_id... truncated ...
```

This part of log should be the output of the method "createTmSample" of org.janelia.jacs2.rest.sync.v2.dataresources.TmSampleResource.
Either ktx data part or RAW data part sets the new sample's FilesystemSync to false, which caused the error in workstation below:
<img width="496" alt="error" src="https://user-images.githubusercontent.com/10544528/71539960-f1594900-297e-11ea-90f6-bb581c025151.png">

**Comment by krokicki** [comment]
Fixed here: https://github.com/JaneliaSciComp/jacs-compute/commit/f946a1a27480f2ec245f5ff0cdd954aebc6bb185
This will be included in the next release (8.18). As a work-around, you can explicitly specify your KTX path at sample creation time.
